lib/routes/token.js

```diff
@@ -89,6 +101,60 @@ function confirmRefreshToken(params) {
   });
 }
 
+function confirmJwt(params) {
+  var assertion = params.assertion;
+  logger.debug('jwt.confirm', assertion);
+
+  return JWTOOL.verify(assertion).catch(function(err) {
+    logger.info('jwt.invalid.verify', err.message);
+    throw AppError.invalidAssertion();
+  }).then(function(payload) {
+    logger.verbose('jwt.payload', payload);
+
+    var client = SERVICE_CLIENTS[payload.iss];
```

Comment:
IIUC, the `iss` field here is the client's opaque hex id.  I think it would be quite neat if the client didn't need to know this value, and could just put their domain-name in directly like `iss: basket.mozilla.org`, in a similar manner to what we do for identity assertions.  One less piece of coupling between the two systems.

That might complicate the data model though.